src/registrar/config/settings.py

```diff
@@ -780,3 +780,11 @@
         # due to Docker, bypass Debug Toolbar's check on INTERNAL_IPS
         "SHOW_TOOLBAR_CALLBACK": lambda _: True,
     }
+
+# From https://django-auditlog.readthedocs.io/en/latest/upgrade.html
+# Run:
+# cf run-task getgov-<> --wait --command 'python manage.py auditlogmigratejson --traceback' --name auditlogmigratejson
+# on our staging and stable, then remove these 2 variables or set to False
+AUDITLOG_TWO_STEP_MIGRATION = True
+
+AUDITLOG_USE_TEXT_CHANGES_IF_JSON_IS_NOT_PRESENT = True
```

Comment:
Yep. The simple way is to make a couple of hotfixes back-to-back to get this guy on staging and stable, but def waiting on @abroddrick  for that.

Reply:
I think it's cleaner to make these environment variables but I won't block on that. I think for what we want this is fine. Instead of doing hotfixes (a change applied on the given release tag). I would just make a follow on PR that needs to be merged after this one gets to staging so they are not in the same release.